include/jemalloc/jemalloc_macros.h.in

```diff
@@ -44,12 +44,12 @@
  *   mallctl("arena." STRINGIFY(MALLCTL_ARENAS_ALL) ".purge", NULL, NULL, NULL,
  *       0);
  */
-#define MALLCTL_ARENAS_ALL	4096
```

Comment:
This is actually kept this way intentionally. See [comments here](https://github.com/jemalloc/jemalloc/blob/5e460bfea25c39d9bf8ea0077c3b6740e9515487/include/jemalloc/jemalloc_macros.h.in#L38)